colorcodemanual.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <assert.h>
+#include "colorcodemanual.h"
+
+void Eachpairofcolors()

```

Comment:
A different person looking at your code might have difficulty in relating the name of method with what it does...
So while choosing name , also think about a later point of time when somebody else is maintaining the code and not us.

Reply:
updated as per review point